app/bundles/CoreBundle/Controller/ThemeController.php

```diff
@@ -206,6 +206,17 @@ public function batchDeleteAction(Request $request, ThemeHelperInterface $themeH
             foreach ($themeNames as $themeName) {
                 $flashes = $this->deleteTheme($themeHelper, $themeName);
             }
+
+            if (count($themeNames) > 1) {
```

Comment:
Please move this to the `ThemeController::deleteTheme` method.

Reply:
hello @shinde-rahul 
no sure how you want me to do it because the deleteTheme method only takes a single theme